test_scripts/WebEngine/WebEngineProjection/008_Registration_reject_WEB_VIEW_app_is_absent_in_PT_WEB_VIEW_part_of_collection.lua

```diff
@@ -0,0 +1,52 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description:
+-- Check that App will be rejected with HMI type WEB_VIEW 
+-- when WEB_VIEW AppHmiType is part of HMI types collection and absent in application policies
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. WebEngine device is connected to SDL
+-- 3. PT contains record for App1 and no record for App2
+
+-- Sequence:
+-- 1. Application1 registers with WEB_VIEW appHMIType successfully
+-- 2. Application2 registers with WEB_VIEW appHMIType successfully
+--  a. SDL rejects registration of application (resultCode DISALLOWED, success:"false")
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared Libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local appSessionId = 1
+local appNotInPTSessionId = 2
+local appHMIType = {"MEDIA", "WEB_VIEW"}
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appHMIType = appHMIType 
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 7
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
+
+config.application2.registerAppInterfaceParams.appHMIType = appHMIType 
+config.application2.registerAppInterfaceParams.syncMsgVersion.majorVersion = 7
+config.application2.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
+
+config.defaultMobileAdapterType = "WS"
+common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Add AppHMIType to preloaded policy table", common.updatePreloadedPT,
+  { appSessionId, appHMIType })
+common.Step("Start SDL, HMI, connect Mobile", common.start)
+
+common.Title("Test")
+common.Step("Register App 1, PT contains record for App1", common.registerAppWOPTU, {appSessionId})
```

Comment:
@YarikMamykin Please add whitespaces before and after table property if it specified in one line:
```suggestion
common.Step("Register App 1, PT contains record for App1", common.registerAppWOPTU, { appSessionId })
```

This comment relates to next scripts too.

Reply:
@aderiabin fixed in https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/32/commits/66e780b7fb2d744f4ac6fd6ee635b18b1bf5b68f